policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,47 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure developmet lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products.
+
+## Policy
+This lifecycle integrates secruity into all aspects of application software development which is comprized of six phases. 
+
+1. Requirements Gathering and Planning:
+- Identify the scope, goals, and necessary resources for development
+- Define functional and security requirements for the feature 
+    - Considering core engineering principles (link)
+    - Security controls such as authentication, data encryption, and secure communication protocols
```

Comment:
It looks like this list is indented so as to align with "1. Requirements Gathering and Planning:" above. Does it make sense to indent it to sit underneath? e.g.:

``` suggestion
  - Identify the scope, goals, and necessary resources for development
  - Define functional and security requirements for the feature 
    - Considering core engineering principles (link)
    - Security controls such as authentication, data encryption, and secure communication protocols